### ISSUE TITLE
Add Timezone support to the new Time region feature

### ISSUE THREAD

**Issue by sanchomuzax** [grafana/grafana#14400]
I know and understand that the new `Time regions` feature in the `Display` menu configuredit refers to UTC time.

Please add dependency to the `General / Time Options / Timezone / Local browser time` setting.

My local timezone is `+0100`.

![2018-12-07 22-12-35](https://user-images.githubusercontent.com/2911588/49674039-cdc01980-fa70-11e8-9dbd-ddf28968fe1e.png)

**Comment by torkelo** [comment]
just convert to UTC before you specify the regions, then it will work in any timezone.

**Comment by sanchomuzax** [comment]
This is not possible in every case.

This is similar problem that #13621 issue.

And I think that if the dasboard is support the timezone feature on the time filter settings then it shuold be required on Time region feature.

**Comment by Gauravshah** [comment]
would like to reopen it. 
@torkelo that does not work for timezones that change offset like PT which changes from +7 to +8

**Comment by AlbanBedel** [comment]
Yes this should be reopened, having the time in UTC will break in all regions that use day light saving.

**Comment by rpwoodbu** [comment]
Daylight Saving Time is another reason to support this. I have graphs that show power usage, annotated by the utility time-of-use schedule. This changes with Daylight Saving Time. Other use cases might include annotating business hours, oncall schedules, etc.

**Comment by ashleydawson00** [comment]
I also have this requirement for annotating shift schedules onto production dashboards.  These change according to daylight savings.

**Comment by hapklaar** [comment]
Why is this closed, is there a solution I missed?

**Comment by tkouba** [comment]
This is not feature request. This is bug in `Time regions`  feature. I cannot use `Time regions` because of daylight saving.

**Comment by mavasaf** [comment]
We are using the dashboard based of from Pacific Time, during daylight savings, the peak hours time region needs to be shifted adjusting to the Day light Start and end. Currently it does not handle this offset.

**Comment by c0yote** [comment]
Need this as well.

**Comment by asamnc** [comment]
Hi, everybody. Do you know something about this?
Iv've been trying to set 2 time regions but with a variable get from one Series in my DB (as a query), but it seems not to work, it always set the 00:00 time. It would be great to set this limits via variables, so you can feed grafana with one DB with all the data and it would automatically set theese values.
Or, another possibility is that in viewer option the final user can change theese values to their needs....
Thanks a lot for your work.

**Comment by AlbanBedel** [comment]
Please re-open, daylight saving is not going away any time soon.

**Comment by AlbanBedel** [comment]
Daylight saving time is not going away, this issue is still relevant.

**Comment by tkouba** [comment]
Daylight saving time is not going away.
